source/blog/2018-07-06-the-ember-times-issue-54.md

```diff
@@ -0,0 +1,131 @@
+---
+title: The Ember Times - Issue No. 54
+author: Jessica Jordan, Alon Bukai, Amy Lam, Kenneth Larsen
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
+alias : "blog/2018/07/06/the-ember-times-issue-54.html"
+responsive: true
+---
+
+привет Emberistas! 🐹
+
+Read either on the [Ember blog](https://www.emberjs.com/blog/2018/07/06/the-ember-times-issue-54.html) or in our [e-mail newsletter](https://the-emberjs-times.ongoodbits.com/2018/07/07/the-ember-times-issue-54) what has been going on in Emberland this week.
+
+...
+
+This week you can read all about the **📦 Ember 3.2** and how the **🐹 Core Team has reorganised**.
+There's also great tips on how you can improve your testing with the new and improved **🚀QUnit DOM**. 
+At last, there's news about **👩‍💻 Twiddle** as well as a nice **📹 video on the Ember Styleguide**. Enjoy!
+
+---
+
+## [Organising the Organisers](https://emberjs.com/blog/2018/06/30/organizing-our-contributors.html)
+The Ember Team has published a new blog post outlining some structural changes to the Core teams. It contains two changes: A new Steering Committee and a renaming of some of the existing teams.
```

Comment:
I'm thinking we could highlight a couple of keywords for each section - what do you think about e.g. 

```
post outlining some **structural changes** to the **Core teams**. [...]
```